insights/parsers/rpm_v_packages.py

```diff
@@ -0,0 +1,54 @@
+"""
+RpmVPackages - command ``/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo``
+==========================================================================================
+"""
+
+from insights.core import CommandParser
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.rpm_V_packages)
+class RpmVPackages(CommandParser):
+    """
+    Class for parsing ``/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo`` command.
+
+    Attributes:
+        packages_list (list of dictionaries): every dictionary contains information about one entry
+
+    Sample output of this command is::
+
+        package procps is not installed
+        ..?......  c /etc/sudoers
+        ..?......    /usr/bin/sudo
+        ..?......    /usr/bin/sudoreplay
+        missing     /var/db/sudo/lectured (Permission denied)
+
+    Examples:
+        >>> type(rpm_v_packages)
+        <class 'insights.parsers.rpm_v_packages.RpmVPackages'>
+        >>> len(rpm_v_packages.packages_list)
+        5
+        >>> sorted(rpm_v_packages.packages_list[0].items())
+        [('attributes', None), ('file', None), ('line', 'package procps is not installed'), ('mark', None)]
+        >>> sorted(rpm_v_packages.packages_list[1].items())
+        [('attributes', '..?......'), ('file', '/etc/sudoers'), ('line', '..?......  c /etc/sudoers'), ('mark', 'c')]
+"""
+
+    def parse_content(self, content):
+        self.packages_list = []
+
+        if len(content) == 0:  # rpm does not report any changes
+            return self.packages_list
```

Comment:
It's OK to remove this `if`, when content is empty, the following `for-loop` will be skipped naturally.
And the caller doesn't handle the return value of the `parse_content`, we do not need to return something in this case.